jvb/src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -1165,7 +1177,18 @@ public IQ healthCheckIqReceived(@NotNull HealthCheckIQ iq)
         /**
          * Number of local endpoints that exist currently.
          */
-        public AtomicLong currentLocalEndpoints = new AtomicLong();
+        public LongGaugeMetric currentLocalEndpoints = METRICS_CONTAINER.registerLongGauge(
+                "current_local_endpoints",
```

Comment:
I think we should, that matches the metric name for the current number of conferences: `"conferences"`.
Should the _variable_ name keep the prefix? For `"conferences"` I named the variable `currentConferences` to avoid ambiguity with the Conference class.